include/drivers/pcie/endpoint/pcie_ep.h

```diff
@@ -273,4 +273,15 @@ int pcie_ep_xfer_data_memcpy(const struct device *dev, uint64_t pcie_addr,
 			     enum pcie_ob_mem_type ob_mem_type,
 			     enum xfer_direction dir);
 
+/**
+ * @brief Data transfer using system DMA
+ *
+ * @details Helper API to achieve data transfer with system DMA
+ *          through PCIe outbound memory
+ */
```

Comment:
parameters and returned value documentation missing

Reply:
This is just wrapper/top level/common public API making use for PCIe EP driver public APIs (such as `pcie_ep_map_addr`, `pcie_ep_unmap_addr`, etc.) for the ease of use. The parameters and returned value documentation is present at the public API level... I don't mind putting it here as well.

Reply:
Added parameters / returned value documentation.